README.md

```diff
@@ -11,9 +11,9 @@ For available `cosign` releases, see https://github.com/sigstore/cosign/releases
 Add the following entry to your Github workflow YAML file:
 
 ```yaml
-uses: sigstore/cosign-installer@v1.3.1
+uses: sigstore/cosign-installer@v1.4.0
```

Comment:
```suggestion
uses: sigstore/cosign-installer@main
```

Since the goal of the action is mostly to verify our release artifacts, it'd be good to encourage consuming the action from `@main` to pick up security updates and instead pinning the version of `cosign` used in the user workload